src/ManagementEnforcer.php

```diff
@@ -656,4 +660,30 @@ public function addFunction(string $name, Closure $func): void
     {
         $this->fm->addFunction($name, $func);
     }
+
+    /**
+     * Gets the index for a given ptype and field.
+     *
+     * @param string $ptype
+     * @param string $field
+     * 
+     * @return int $fieldIndex
+     * @throws Exceptions\CasbinException
+     */
+    public function getFieldIndex(string $ptype, string $field): int
+    {
+        return $this->model->getFieldIndex($ptype, $field);
```

Comment:
This function is not covered by unit tests
see: https://coveralls.io/builds/68082308/source?filename=src%2FManagementEnforcer.php#L675